### ISSUE TITLE
Error in Ctrl+T dialog

### ISSUE THREAD

**Issue by EnCey** [careteditor/caret#430]
As mentioned in #427 the file dialog (Ctrl+T) often doesn't show all files or no files at all.

After navigating to a different file via Ctrl+1 (because Ctrl+T didn't show any files), I tried Ctrl+T again and the following error occurred:
```
{
    "message": "Cannot read property 'isDirectory' of undefined",
    "version": "3.1.0",
    "os": "Windows 10",
    "path": "C:\\Users\\myname\\AppData\\Local\\Caret\\app-3.1.0\\resources\\app.asar\\packages\\Project\\Project.js",
    "code": "parent:e,name:t,fullPath:r,type:i.isDirectory()?\"dir\":i.isFile()?\"file\":void 0}",
    "line": 5,
    "column": 4897,
    "stack": [
        "at t.value ((C:\\Users\\myname\\AppData\\Local\\Caret\\app-3.1.0\\resources\\app.asar\\packages\\Project\\Project.js:5:4897)",
        "at t.value ((C:\\Users\\myname\\AppData\\Local\\Caret\\app-3.1.0\\resources\\app.asar\\packages\\Project\\Project.js:5:5049)",
        "at t.value ((C:\\Users\\myname\\AppData\\Local\\Caret\\app-3.1.0\\resources\\app.asar\\packages\\Project\\Project.js:5:4602)",
        "at C:\\Users\\myname\\AppData\\Local\\Caret\\app-3.1.0\\resources\\app.asar\\packages\\Project\\Project.js:5:4352",
        "at FSReqWrap.oncomplete ((fs.js:114:15)"
    ]
}
```

Some more info:
when I launch Caret it typically opens `C:\Dropbox\notes\cbn\working.md`, which is more often than not my most recently edited file. No special characters or so in the path.

I currently have 5 other folders in the `cbn` folder, all containing mostly `.md` files but also some others (images, word documents, etc.).  No special characters in any file or folder names as far as I can tell, the most unusual ones are my meeting documents, which have a date prefix, e.g. `17-03-29-PreDesignMeeting.md`.

Any other information I should provide? The actual file names and contents I don't really want to expose as they're work related.

**Comment by astoilkov** [comment]
Thanks for reporting. We have investigated and fixed this issue. The next release will include the fix.

**Comment by astoilkov** [comment]
This issue is now fixed in the latest Caret 3.2 version.

Closing this issue. Thanks for reporting.
